app/products/calls/connection/connection.ts

```diff
@@ -68,15 +74,48 @@ export async function newConnection(
         }
 
         try {
-            stream = await mediaDevices.getUserMedia({
+            voiceStream = await mediaDevices.getUserMedia({
                 video: false,
                 audio: true,
             }) as MediaStream;
-            voiceTrack = stream.getAudioTracks()[0];
+            voiceTrack = voiceStream.getAudioTracks()[0];
             voiceTrack.enabled = false;
-            streams.push(stream);
+            streams.push(voiceStream);
         } catch (err) {
-            logError('calls: unable to get media device:', err);
+            logError('calls: unable to get audio stream:', err);
+        }
+    };
+
+    const initializeVideoTrack = async (start: boolean) => {
+        if (videoTrack) {
+            return;
+        }
+
+        try {
+            videoStream = await mediaDevices.getUserMedia({
+                video: {
+                    frameRate: 30,
+                    facingMode: 'user', // TODO: allow user to switch camera
+                },
```

Comment:
We need the UX for this, but eventually, we should be able to switch to/from front/rear camera.